### ISSUE TITLE
The trainning of model for Text Detecting

### ISSUE THREAD

**Issue by Iric2018** [yangxue0827/R2CNN_FPN_Tensorflow#43]
Hi, thanks for your share of repo. When I run the train1.py ，the output of the paraof the parameters of the pretrained model and 'fast_rcnn_loc_loss' , 'fast_rcnn_loc_rotate_loss'  are  zero all the time. Are there any problems? The output are show as following:

resnet_v1_101/block4/unit_2/bottleneck_v1/conv3/BatchNorm/gamma:0
resnet_v1_101/block4/unit_2/bottleneck_v1/conv3/BatchNorm/beta:0
resnet_v1_101/block4/unit_2/bottleneck_v1/conv3/BatchNorm/moving_mean:0
resnet_v1_101/block4/unit_2/bottleneck_v1/conv3/BatchNorm/moving_variance:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/weights:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/BatchNorm/gamma:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/BatchNorm/beta:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/BatchNorm/moving_mean:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/BatchNorm/moving_variance:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/weights:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/BatchNorm/gamma:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/BatchNorm/beta:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/BatchNorm/moving_mean:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/BatchNorm/moving_variance:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/weights:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/BatchNorm/gamma:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/BatchNorm/beta:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_mean:0
resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_variance:0
restore model
 2018-06-08 15:06:45: step1    image_name:OCR_img_140048.jpg |	
                                rpn_loc_loss:264.442718506 |	 rpn_cla_loss:0.828770041466 |	
                                rpn_total_loss:265.271484375 |
                                fast_rcnn_loc_loss:0.0 |	 fast_rcnn_cla_loss:2.29521894455 |	
                                fast_rcnn_loc_rotate_loss:0.0 |	 fast_rcnn_cla_rotate_loss:1.15461230278 |	
                                fast_rcnn_total_loss:3.44983124733 |	
                                total_loss:269.557434082 |	 pre_cost_time:3.70543599129s
 2018-06-08 15:06:53: step11    image_name:OCR_img_143147.jpg |	
                                rpn_loc_loss:167.275375366 |	 rpn_cla_loss:0.0647400766611 |	
                                rpn_total_loss:167.340118408 |
                                fast_rcnn_loc_loss:0.0 |	 fast_rcnn_cla_loss:0.655150413513 |	
                                fast_rcnn_loc_rotate_loss:0.0 |	 fast_rcnn_cla_rotate_loss:0.657649993896 |	
                                fast_rcnn_total_loss:1.31280040741 |	
                                total_loss:169.489440918 |	 pre_cost_time:0.217247009277s
 2018-06-08 15:06:56: step21    image_name:OCR_img_10706.jpg |	
                                rpn_loc_loss:203.170288086 |	 rpn_cla_loss:0.166537106037 |	
                                rpn_total_loss:203.33682251 |
                                fast_rcnn_loc_loss:0.0 |	 fast_rcnn_cla_loss:0.572579860687 |	
                                fast_rcnn_loc_rotate_loss:0.0 |	 fast_rcnn_cla_rotate_loss:0.579669415951 |	
                                fast_rcnn_total_loss:1.15224933624 |	
                                total_loss:205.327468872 |	 pre_cost_time:0.273305892944s
 2018-06-08 15:06:59: step31    image_name:OCR_img_143361.jpg |	
                                rpn_loc_loss:252.477355957 |	 rpn_cla_loss:0.028674390167 |	
                                rpn_total_loss:252.506027222 |
                                fast_rcnn_loc_loss:0.0 |	 fast_rcnn_cla_loss:0.0728850066662 |	
                                fast_rcnn_loc_rotate_loss:0.0 |	 fast_rcnn_cla_rotate_loss:0.0746394395828 |	
                                fast_rcnn_total_loss:0.147524446249 |	
                                total_loss:253.493240356 |	 pre_cost_time:0.2667491436s
 2018-06-08 15:07:02: step41    image_name:OCR_img_141126.jpg |	
                                rpn_loc_loss:212.294830322 |	 rpn_cla_loss:0.0308034606278 |	
                                rpn_total_loss:212.325637817 |
                                fast_rcnn_loc_loss:0.0 |	 fast_rcnn_cla_loss:0.41481667757 |	
                                fast_rcnn_loc_rotate_loss:0.0 |	 fast_rcnn_cla_rotate_loss:0.430022597313 |	
                                fast_rcnn_total_loss:0.844839274883 |	
                                total_loss:214.012130737 |	 pre_cost_time:0.226809024811s
 2018-06-08 15:07:04: step51    image_name:OCR_img_1434319.jpg |	
                                rpn_loc_loss:170.955886841 |	 rpn_cla_loss:0.0378239601851 |	
                                rpn_total_loss:170.993713379 |
                                fast_rcnn_loc_loss:0.0 |	 fast_rcnn_cla_loss:0.33990368247 |	
                                fast_rcnn_loc_rotate_loss:0.0 |	 fast_rcnn_cla_rotate_loss:0.357002526522 |	
                                fast_rcnn_total_loss:0.696906208992 |	
                                total_loss:172.533584595 |	 pre_cost_time:0.259843826294s
 2018-06-08 15:07:07: step61    image_name:OCR_img_142783.jpg |	
                                rpn_loc_loss:227.995605469 |	 rpn_cla_loss:0.0456900112331 |	
                                rpn_total_loss:228.041290283 |
                                fast_rcnn_loc_loss:0.0 |	 fast_rcnn_cla_loss:0.0201326087117 |	
                                fast_rcnn_loc_rotate_loss:0.0 |	 fast_rcnn_cla_rotate_loss:0.0213839225471 |	
                                fast_rcnn_total_loss:0.0415165312588 |	
                                total_loss:228.926528931 |	 pre_cost_time:0.291739940643s
 2018-06-08 15:07:10: step71    image_name:OCR_img_phone_1029.jpg |	
                                rpn_loc_loss:168.393371582 |	 rpn_cla_loss:0.0498738214374 |	
                                rpn_total_loss:168.443252563 |
                                fast_rcnn_loc_loss:0.0 |	 fast_rcnn_cla_loss:0.247651547194 |	
                                fast_rcnn_loc_rotate_loss:0.0 |	 fast_rcnn_cla_rotate_loss:0.265152215958 |	
                                fast_rcnn_total_loss:0.512803792953 |	
                                total_loss:169.801071167 |	 pre_cost_time:0.253770112991s

**Comment by yangxue0827** [comment]
keep training and open tensorboard
